enigma-swing/src/main/java/cuchaz/enigma/gui/util/TranslationChangeListener.java

```diff
@@ -0,0 +1,7 @@
+package cuchaz.enigma.gui.util;
+
+public interface TranslationChangeListener {
```

Comment:
Maybe this should be like `LanguageChangeListener`? Might be a bit confusing since enigma also deals with source translation.

Reply:
Good point, yeah.